Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs

```diff
@@ -234,13 +240,15 @@ protected void AddToValidators (int columnIndex, IEnumerable<BaseValidator> vali
     {
       ArgumentUtility.CheckNotNull ("validators", validators);
 
-      ControlCollection cellValidators = GetValidators (columnIndex);
+      ControlCollection? cellValidators = GetValidators (columnIndex);
```

Comment:
I think we should add an assertion here and then we can drop the `!` two lines down.